### PR TITLE
Added ExposedResponse to LocationTracking screen

### PR DESCRIPTION
**Pull request by adnanjt**
#### Description:
Include an advice layout for DATA MATCHED users. When you are Exposed it has a button to see the next steps and this shows Response 5 from the CDC. 

https://trello.com/c/yJEvQlP8/90-include-an-advice-layout-for-data-matched-users

#### Screenshots:

image

image

image

#### How to test:

1 - Ir a la pantallla de Feture flash a travez de settings para activar el exposed screen para prueba. 

2 - A travez de Home -> Comprobar contacto entras a la pagina de exposed

3 - Aqui debe haber un boron nuevo que dice "Pasos a seguir" si pulsas esto te lleva a una pagina de informacion para las personas expuesta la respuesta 5

### CODE REVIEW

File: app/components/DR/LocationTracking/NextSteps.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+class NextSteps extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let buttonLabel;
+    let buttonFunction;
+    //buttonLabel = languages.t('label.see_exposure_history');
```
Comment (JoseFcoEstrella) [inline app/components/DR/LocationTracking/NextSteps.js]: Why do you keep this?

Comment (adnanjt) [reply app/components/DR/LocationTracking/NextSteps.js]: Thanks this can be delted will do it

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM

File: app/components/DR/LocationTracking/NextSteps.js [hunk 2]
```diff
@@ -29,7 +30,7 @@ class NextSteps extends Component {
 const styles = StyleSheet.create({
   buttonContainer: {
     marginTop: 24,
-    marginBottom: 24,
+    //marginBottom: 24,
```
Comment (inteljudajocu) [inline app/components/DR/LocationTracking/NextSteps.js]: Remove comment if this line is unnecessary

**Review by inteljudajocu — COMMENTED** [review]
Comments

File: app/components/DR/LocationTracking/NextSteps.js (hunk 2 above)
Comment (adnanjt) [reply app/components/DR/LocationTracking/NextSteps.js]: Good good

**Comment by adnanjt** [comment]
@inteljudajocu  Ready para el approve?

**Comment by adnanjt** [comment]
@inteljudajocu  @JoseFcoEstrella  Probaron el screen ?? del exposed ? que se vea y los links internos ?

**Review by inteljudajocu — APPROVED** [review]
LGTM, @adnanjt it was tested

**Comment by adnanjt** [comment]
thanks @inteljudajocu  Im going to take out some of the testing screens then and merging

**Pull request merged**